util/xfunc.h

```diff
@@ -1,113 +0,0 @@
-//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
-//  This source code is licensed under the BSD-style license found in the
-//  LICENSE file in the root directory of this source tree. An additional grant
-//  of patent rights can be found in the PATENTS file in the same directory.
-#pragma once
-
-#include <cstdlib>
-#include <functional>
-#include <string>
-
-#include "rocksdb/options.h"
-
-namespace rocksdb {
-
-/*
- * If ROCKSDB_XFTEST_FORCE has a value of 1, XFUNC is forced to be defined.
- * If ROCKSDB_XFTEST_FORCE has a value other than 1,
- *    XFUNC is forced to be undefined.
- * If ROCKSDB_XFTEST_FORCE is undefined, XFUNC is defined based on NDEBUG,
- *   with XFUNC only being set for debug builds.
- */
-#if defined(ROCKSDB_XFTEST_FORCE)
```

Comment:
I don't understand this. This feature doesn't exist anymore, so the flags shouldn't be consumed. Is setting the flags still important?